obp/dataset/synthetic_multi.py

```diff
@@ -20,7 +20,7 @@
 
 
 @dataclass
-class SyntheticBanditDatasetWithMultiLoggers(SyntheticBanditDataset):
+class SyntheticMultiLoggersBanditDataset(SyntheticBanditDataset):
```

Comment:
[nits]

`rhos: List[float]` should be `rhos: List[Union[int, float]]` (comment)